CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Added
 
+- Self-hosted etcd will mount a hostPath volume directory defined as
+  value of annotation `selfhosted-etcd.alpha.coreos.com/checkpointdir`.
```

Comment:
as mentioned below. i do not think we need this. the current hostpath of etcd is a well known and not configurable place. we should just checkpoint into that hostpath directly. if we want to make the dir configurable, we should make etcd dir configurable first, and then still put checkpoint there.